conf/ConfigOptions.asciidoc

```diff
@@ -1173,6 +1174,8 @@ this to true can be useful when conforming to strict specifications.
 * Data Type: string
 * Default Value: ``
 
+TODO: description
```

Comment:
No big deal here but we should add a description.  It shouldn't hold up merging the pull request because of the phantom Jenkins failures.